backend/api/Areas/Admin/Controllers/ParcelController.cs

```diff
@@ -128,53 +123,72 @@ public IActionResult GetParcelByPID(string pid)
         /// <param name="model">The parcel model.</param>
         /// <returns>The parcel added.</returns>
         [HttpPost]
-        public IActionResult AddParcel([FromBody] ParcelModel model)
+        public IActionResult AddParcel ([FromBody] ParcelModel model)
         {
-            var entity = _mapper.Map<Entity.Parcel>(model);
-            var userId = this.User.GetUserId();
+            var entity = _mapper.Map<Entity.Parcel> (model);
+            entity.Evaluations.Add (new Entity.ParcelEvaluation (model.FiscalYear, entity) // TODO: Move this logic to AutoMapper.
+                {
+                    EstimatedValue = model.EstimatedValue,
```

Comment:
This spacing is kinda weird, but I don't see the value in fixing it if we automapper this instead.

Reply:
Again another auto-formatting that is driving me crazy...  Honestly don't know why anyone would want that.  But I can't find the one causing it.

Reply:
Think I will get rid of some of my formatters in vscode.